src/server/paths/api/v0/character.js

```diff
@@ -24,7 +24,7 @@ class Character {
       return;
     }
 
-    res.type('application/json').send(jsonSerialize(character));
+    res.type('application/json').send(jsonSerialize(character, { isJSON: true }));
```

Comment:
We intended to use `serialize-javascript` as an anti-XSS form of JSON serialization, not to serialize functions. (If anything, serializing functions *exposes* security holes.) Accordingly, use the `isJSON` option to only serialize into standard JSON.